src/components/InventoryGroups/Modals/AddSystemsToGroupModal.js

```diff
@@ -92,14 +106,75 @@ const AddSystemsToGroupModal = ({
     dispatch(clearFilters());
   };
 
+  const edgeParityEnabled =
```

Comment:
This might raise an error, because hooks are called conditionally. 
It should be more something like this:

```js

const edgeInventoryListEnabled = useFeatureFlag('edgeParity.inventory-list');
const edgeInventoryGroups = useFeatureFlag('edgeParity.inventory-groups-enabled');
const edgeParityEnabled = edgeParityIsAllowed && edgeInventoryListEnabled && edgeInventoryGroups;

```

Reply:
Thanks @bastilian for pointing this, fixed